src/main/java/org/jenkinsci/plugins/workflow/cps/CpsWhitelist.java

```diff
@@ -103,6 +104,9 @@ public boolean permitsStaticMethod(Method method, Object[] args) {
             String driverTo = "org.codehaus.groovy.runtime.";
             if (cn.startsWith(driverFrom)) {
                 try {
+                    if (Modifier.isPrivate(method.getModifiers())) {
+                        return true;
```

Comment:
Ok, I just spent way too long trying to figure this out. We are definitely generating CPS code, so, for example, `CpsDefaultGroovyMethods.$collectEntries__java_util_Iterator__java_util_Map__groovy_lang_Closure` is calling `$addEntry__java_util_Map__java_lang_Object` via `b.staticCall(...)`. Which, I think, means that `ContinuationGroup.methodCall(...)` is getting called at runtime, and *that* calls `Invoker.methodCall(...)`, which is resulting in `SandboxInvoker.methodCall(...)` and hence the rest of the stacktrace. I did some tests and I don't think `SandboxTransformer` is ever being invoked for this method, I think it's just that the `Invoker` is `SandboxInvoker` and so tada? I dunno. A theory.